src/main/java/io/fabric8/maven/docker/access/UrlBuilder.java

```diff
@@ -81,7 +86,7 @@ public String inspectExecContainer(String containerId) {
     }
 
     public String listContainers(String ... filter) {
-        Builder builder = u("containers/json");
+        Builder builder = u("containers/json").p("all", true);
```

Comment:
Not sure why do need "all" here ?

Reply:
Yes, this is so even if the container is stopped it can still be removed.  Without "all" only running containers are returned.